TicTacToeProject/Board.cs

```diff
@@ -15,6 +15,14 @@ public Board()
             }
         }
 
+        public void partiallyFillBoard(int[] moves, string marker)
```

Comment:
I would put something like this on a test helper -- so you could do something like
```
BoardManipulator.partiallyFillBoard(board, marker, moves)
```
without leaving behind test-only code in your production code.